Coding/fw/lib/WebServer/Web_Config.cpp

```diff
@@ -0,0 +1,95 @@
+/***************************************************************************************************
+  (c) NewTec GmbH 2020   -   www.newtec.de
+  $URL: https://github.com/NewTec-GmbH/esp32-can-iot $
+***************************************************************************************************/
+/**
+@addtogroup Webserver
+@{
+@file       Web_Config.cpp
+
+Stores the Credentials and configuration of the Webserver. @ref Web_config.h
+
+* @}
+***************************************************************************************************/
+/* INCLUDES ***************************************************************************************/
+#include "Web_Config.h"
+
+/* C-Interface ************************************************************************************/
+extern "C"
+{
+}
+
+/* CONSTANTS **************************************************************************************/
+static const String DIRECTORY = "Server";
+static String STA_SSID = "";
+static String STA_PASSWORD = "";
+static String AP_SSID = "ESP32";
+static String AP_PASSWORD = "hochschuleulm";
+static String WEB_USER = "admin";
+static String WEB_PASSWORD = "admin";
+
+/* MACROS *****************************************************************************************/
+
+/* TYPES ******************************************************************************************/
+
+/* PROTOTYPES *************************************************************************************/
+
+/* VARIABLES **************************************************************************************/
+
+/* PUBLIC METHODES ********************************************************************************/
+/**************************************************************************************************/
+
+void WebConfig::importConfig()
+{
+    Settings::get(DIRECTORY, "STA_SSID", STA_SSID, "");
+    Settings::get(DIRECTORY, "STA_Password", STA_PASSWORD, "");
+}
+
+/**************************************************************************************************/
+String &WebConfig::getSTA_SSID()
```

Comment:
all these return need to be const 
place the & after String like String&. It belongs to the type, not the function